src/simplex.ts

```diff
@@ -166,24 +261,24 @@ export class Simplex {
 
     // Various "moves"
     private _reflect(point: Point, centroid: number[]) {
-        return this._point(weightedSum(Simplex.rho, centroid, point.location));
+        return this._point(weightedSum(this.rho, centroid, point.location));
     }
 
     private _expand(point: Point, centroid: number[]) {
-        return this._point(weightedSum(Simplex.chi, centroid, point.location));
+        return this._point(weightedSum(this.chi, centroid, point.location));
     }
 
     private _contractInside(point: Point, centroid: number[]) {
-        return this._point(weightedSum(Simplex.psi, centroid, point.location));
+        return this._point(weightedSum(this.psi, centroid, point.location));
     }
 
     private _contractOutside(point: Point, centroid: number[]) {
-        return this._point(weightedSum(-Simplex.psi * Simplex.rho,
+        return this._point(weightedSum(-this.psi * this.rho,
                                        centroid, point.location));
     }
 
     private _shrink(point: Point, best: number[]) {
-        return this._point(weightedSum(-Simplex.sigma, best, point.location));
+        return this._point(weightedSum(-this.sigma, best, point.location));
```

Comment:
yeah, this is the same coverage hole as the initial PR (https://github.com/reside-ic/dfoptim/pull/1) but flagging up here because I touched the code by moving these from being public static fields to private readonly fields.

There's a "move" called shrinking that is unlikely in practice and I've not managed to find a nice test case that will trigger it. It's something to do at some point for sure